layouts/post/single.html

```diff
@@ -21,9 +21,6 @@ <h1 class="post-title">{{ .Title }}</h1>
         <a class="share share-twitter" target="_blank" href="https://twitter.com/intent/tweet/?text={{ .Title }}&amp;url={{ .Permalink }}" title="Share on Twitter">
           <svg class="icon icon-twitter" fill="currentColor"><use xlink:href="#icon-twitter" /></svg>
         </a>
-        <a class="share share-google" target="_blank" href="https://plus.google.com/share?url={{ .Title }}&amp;url={{ .Permalink }}" title="Share on Google+">
-          <svg class="icon icon-google" fill="currentColor"><use xlink:href="#icon-googleplus" /></svg>
-        </a>
```

Comment:
Please remove Goggle+ icon symbol, also class name in sass too, thanks!

Reply:
Check out the following changes, thanks!